### PR TITLE
Fix memory leak in `nsstring_from_str`

### PR DESCRIPTION
**Pull request by aleksijuvani**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -47,10 +47,8 @@ fn nsstring_from_str(string: &str) -> *mut objc::runtime::Object {
     let cls = class!(NSString);
     let bytes = string.as_ptr() as *const c_void;
     unsafe {
-        let obj: *mut objc::runtime::Object = msg_send![cls, alloc];
-        let obj: *mut objc::runtime::Object = msg_send![obj, initWithBytes:bytes
-                                                    length:string.len()
-                                                    encoding:UTF8_ENCODING];
+        let mut obj: *mut objc::runtime::Object = msg_send![cls, alloc];
+        obj = msg_send![obj, initWithBytes:bytes length:string.len() encoding:UTF8_ENCODING];
```
Comment (kvark) [inline src/lib.rs]: how is this fixing anything? your change looks equivalent to the old code

Comment (aleksijuvani) [reply src/lib.rs]: My bad! I was seeing a memory leak show up in Instruments pointing to this function, and it disappeared upon making this change. However, upon further inspection, it looks like this was actually already fixed by #85, which hasn't been published yet. I had assumed that gfx-backend-metal was using the latest commit. I thought I had tested this before making the change, but I must've had the dependency pointing at the wrong version at the time.

Comment (aleksijuvani) [reply src/lib.rs]: I would suggest publishing a new version of this crate so that we can take advantage of the latest changes in gfx-backend-metal.

Comment (kvark) [reply src/lib.rs]: I'd like to release, but it looks like #89 needs to be resolved first in some way

**Pull request closed without merging**